src/docs/product/issues/suspect-commits/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Suspect Commits
+sidebar_order: 2
```

Comment:
This doesn't really seem like the correct position for this under Issues. I recommend it either be made a child page of _Issues States & Triage_ or come after that page at the same level if you're concerned it'll be too buried underneath it.
```suggestion
sidebar_order: 2
```